### PR TITLE
LP-164-Improve UI components

### PR DESCRIPTION
**Pull request by melissaquezada94**
Title: LP-164 Improve UI components
Description: Improve UI components to make them similar to the UI
Link: https://devaptivist.atlassian.net/browse/LP-164
Files:
Screenshot 2023-04-26 at 3 16 24 PM
Screenshot 2023-04-26 at 3 16 45 PM
Screenshot 2023-04-26 at 4 40 17 PM
Screenshot 2023-04-26 at 5 11 58 PM
Screenshot 2023-04-26 at 5 12 07 PM
Screenshot 2023-04-26 at 5 16 48 PM
Screenshot 2023-04-26 at 5 16 57 PM
Screenshot 2023-04-26 at 5 22 06 PM
Screenshot 2023-04-26 at 5 22 21 PM
Screenshot 2023-04-26 at 5 24 15 PM
Screenshot 2023-04-26 at 5 24 40 PM
Screenshot 2023-04-26 at 5 28 29 PM
Screenshot 2023-04-26 at 5 28 40 PM
Screenshot 2023-04-26 at 5 31 45 PM
Screenshot 2023-04-26 at 5 39 38 PM
Screenshot 2023-04-26 at 5 53 55 PM

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 1]
```diff
@@ -170,23 +169,22 @@ fun LpPostCard(
...
                 color = MaterialTheme.colorScheme.onPrimaryContainer,
                 fontSize = 15.sp
             )
             Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.End) {
-                IconButton(onClick = onCommentClick) {
+                IconButton(onClick = onCommentClicked) {
                     Icon(
                         Icons.Outlined.Comment,
-                        "Comment",
+                        stringResource(R.string.content_description_comment),
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: also here the name params

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 2]
```diff
@@ -288,27 +286,26 @@ fun LPGenericElevatedCard(
...
-            Spacer(modifier = Modifier.height(LeasePertTheme.sizes.extraSize6))
             Text(
                 text = description,
                 style = LPTypography.bodyMedium,
                 color = colors.inversePrimary,
                 maxLines = 3,
                 overflow = TextOverflow.Ellipsis,
                 textAlign = TextAlign.Justify,
-                fontSize = 14.sp
+                fontSize = 14.sp,
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: I think we don't need to add this fontSize cause style already has it

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Chips.kt [hunk 3]
```diff
@@ -43,7 +47,7 @@ fun LpGenericChip(
             .padding(
                 all = sizes.midSmallSize,
             )
-            .clickable { onClick },
+            .clickable(enabled = true, onClick = onClicked),
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Chips.kt]: I think "enable = true" is not necessary cause it's true by default

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 4]
```diff
@@ -121,36 +123,38 @@ fun LpFilledTonalButton(
...
             .padding(LeasePertTheme.sizes.smallSize)
             .height(LeasePertTheme.sizes.extraSize48)
             .width(LeasePertTheme.sizes.extraSize48),
-        containerColor = color,
-        contentColor = colors,
+        containerColor = containerColor,
         elevation = elevation
     ) {
-        Icon(Icons.Outlined.Edit, "edit button", tint = MaterialTheme.colorScheme.onPrimary)
+        Icon(icon, contentDescription, tint = iconColor)
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: let's add the name params

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 2 above)
Comment (melissaquezada94) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: alright!

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Chips.kt (hunk 3 above)
Comment (melissaquezada94) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Chips.kt]: thanks!

**Review by carlosph816 — APPROVED** [review]
Well done!

**Review by irv205 — APPROVED** [review]

**Pull request merged**